### DISCUSSION TITLE
State of open source Whisper GUIs?

### DISCUSSION THREAD

**Discussion by eloquence** [openai/whisper#2180] [Q&A]
It looks like the most mature Whisper GUIs are currently proprietary or partially so:
- [MacWhisper](https://github.com/openai/whisper/discussions0) is proprietary, with a free base version
- [WhisperScript](https://getwavery.gumroad.com/l/whisperscript) is proprietary, with a free lite version
- [Buzz](https://github.com/chidiwilliams/buzz) is open source and cross-platform, but the [Apple Silicon version](https://apps.apple.com/us/app/buzz-captions/id6446018936?ign-itscg=30200&ign-itsct=apps_box_badge) is proprietary 

(Please correct me if I'm wrong about any of those, just going by project descriptions and such.)

Among the open source efforts, [Whiskey](https://github.com/bits-by-brandon/whisper-ui)  looks the most promising but is still very incomplete (I say this with gratitude and appreciation for the maintainer!). There are also [Stage-Whisper](https://github.com/Stage-Whisper/Stage-Whisper) (inactive for a while now) and a few Tkinter UIs.

Journalists and human rights defenders would benefit from a reliable, mature, cross-platform, open source GUI for AI-based transcription. Offline transcription is really important to mitigate various risks journalists face when dealing with sensitive audio and video recordings. Sure, a $10 Mac app is not a huge expense, but open source also opens more avenues for cross-platform support, and can help ensure a project's longevity.

TANSTAAFL, but  I do think such a project would be fundable via philanthropic supporters in this space. But before exploring such avenues, I was wondering if I'm missing any existing open source projects that already have significant traction that it might be worth reaching out to, in addition to the above. 

tl;dr - did I miss any mature open source GUIs for Whisper?

**Comment by gizmo84** [comment]
Vibe: https://github.com/thewh1teagle/vibe

**Comment by eloquence** [comment]
Ooh, Vibe looks very promising as well. Just gave it a quick spin and it worked immediately (with one false start when the model download failed). Thanks much for the pointer.

**Comment by phineas-pta** [comment]
https://github.com/search?q=whisper%20gui&type=repositories

i see at least 100 projects to try out, but if u use mac the number maybe less
